runtime/JavaScript/src/antlr4/atn/ATN.js

```diff
@@ -139,4 +139,5 @@ ATN.prototype.getExpectedTokens = function( stateNumber, ctx ) {
 
 ATN.INVALID_ALT_NUMBER = 0;
 
-exports.ATN = ATN;
\ No newline at end of file
+exports.ATN = ATN;
+exports.INVALID_ALT_NUMBER = ATN.INVALID_ALT_NUMBER;
```

Comment:
📝 Per https://github.com/antlr/antlr4/pull/2496#discussion_r257440907, it seems like this change should not be necessary

Reply:
`RuleContext.js` & `tree/Trees.js` import `INVALID_ALT_NUMBER` as

`var INVALID_ALT_NUMBER = require('<path>/atn/ATN').INVALID_ALT_NUMBER`;

They are looking for `INVALID_ALT_NUMBER` to be directly on the exports object.

For some reason Node is not okay with
`var INVALID_ALT_NUMBER = require('<path>/atn/ATN').ATN.INVALID_ALT_NUMBER`;. Thinks ATN in here is undefined.